Makefile

```diff
@@ -21,24 +24,7 @@ init-symfony:
 init: build up init-symfony
 
 rm:
-	docker compose exec apache sh -c \
-	"rm -rf \
-	./assets \
-	./bin \
-	./config \
-	./migrations \
-	./public/index.php \
-	./src \
-	./templates \
-	./tests \
-	./translations \
-	./var \
-	./vendor \
-	./composer.json \
-	./composer.lock \
-	./symfony.lock \
-	./phpunit.xml.dist \
-	./.env \
-	./.env.test \
-	./.gitignore \
-	./importmap.php"
+	docker compose down -v
+
+prune:
+	docker system prune -f --volumes
```

Comment:
I believe there may have been a misunderstanding regarding the intended purpose of the `rm` command. My objective was to gain the ability to swiftly remove newly created Symfony files, thereby expediting my testing workflow. This was not intended to remove containers.

The workflow I envisioned would proceed as follows:

1. Initiate a new Symfony project.
2. Occasionally, this process may not be successful.
3. In such instances, I would execute the `make rm` command to automatically remove Symfony project files, eliminating the need for manual intervention.
4. Modify the project files as required.
5. Initiate a new project to ascertain whether the modifications have been effective.

Given these considerations, I would like to restore my `rm` command. However, I acknowledge the existence of your `rm` command. To facilitate this, it would be beneficial to rename one of the commands to avoid potential conflicts.